CHANGELOG.md

```diff
@@ -23,6 +23,33 @@ CHANGELOG
 
 </details>
 
+Swift Next
+----------
+
+* [SE-0269][]:
+
+  When an escaping closure explicitly captures `self` it its capture list, the
```

Comment:
Nit: should be “in its capture list”, not “it its”

Reply:
Ugh I completely missed this! Thank you for spotting it!

Reply:
😱 good catch, thanks Brent!
```suggestion
  When an escaping closure explicitly captures `self` in its capture list, the
```